modules/working-in-code-server.adoc

```diff
@@ -0,0 +1,67 @@
+
+For more information on code-server, see link:https://github.com/coder/code-server[code-server in GitHub].
+
+NOTE: Elyra-based pipelines are not available with the code-server workbench image. 
+
+
+ifndef::upstream[]
+[IMPORTANT]
+====
+ifdef::self-managed[]
+The code-server notebook image is currently available in {productname-long} {vernum} as a Technology Preview feature.
+endif::[]
+ifdef::cloud-service[]
+The code-server notebook image is currently available in {productname-long} as a Technology Preview feature.
+endif::[]
+Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete.
+Red{nbsp}Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+endif::[]
+
+
+== Installing extensions with code-server
+
+With the code-server workbench image, you can customize your code-server environment by adding extensions that add new languages, themes, debuggers, and connect to additional services. You can also enhance the efficiency of your data science work with extensions for syntax highlighting, auto-indentation, and bracket matching. 
+
+For details about the third-party extensions that you can install with code-server, see the link:https://open-vsx.org/[Open VSX Registry] which provides extensions for VS Code compatible editors.
+
+
+.Prerequisites
+
+* You are logged in to {productname-long}.
+ifndef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have created a data science project that has a code-server workbench.
+
+.Procedure
+
+. From the {productname-short} dashboard, click *Data Science Projects*.
++
+The *Data Science Projects* page opens.
+. Click the name of the project containing the code-server workbench you want to start.
++
+A project details page opens.
+. Click the *Workbenches* tab.
+. Click the toggle in the *Status* column for the relevant workbench to start a workbench that is not running.
++
+The status of the workbench that you started changes from *Stopped* to *Running*.
+. After the workbench has started, click *Open* to open the workbench notebook.
+. In the Activity Bar, click the Extensions icon. (image:images/codeserver-extensions-icon.png[The Extensions icon])
```

Comment:
Do we usually reference icons in this order?
In the Activity Bar, click the Extensions (image:images/codeserver-extensions-icon.png[The Extensions icon]) icon.

Reply:
this is in the code-server UI. I made a minor fix to the text.